DEPS

```diff
@@ -1,26 +1,29 @@
 use_relative_paths = True
 
 deps = {
-  "vendor/requests": "https://github.com/kennethreitz/requests@e4d59bedfd3c7f4f254f4f5d036587bcd8152458",
+  "components/playlist/browser/resources/media_detector": {
+    "url": "https://github.com/brave/playlist-component.git@5434730bf7342f1ba5c057f1640882bb38604a85",
```

Comment:
I don't like that we have `media_detector` directory, but `playlist-component` repository. All other deps seem to be more consistent.

Reply:
Yeah, playlist-component would be better dir name.